cli/src/config.rs

```diff
@@ -451,6 +450,10 @@ fn env_base_layer() -> ConfigLayer {
         .inspect_err(|err| tracing::warn!(?err, "failed to get username"))
     {
         layer.set_value(OP_USERNAME, value).unwrap();
+    } else if let Ok(value) = env::var("USER") {
+        // On Unix, $USER is set by login(1). Use it as a fallback because
+        // getpwuid() of musl libc appears not (fully?) supporting nsswitch.
+        layer.set_value(OP_USERNAME, value).unwrap();
```

Comment:
If this is a last resort, shouldn't we even use something like

```rust
  if let Some(value) = env::var_os("USER") {
    layer.set_value(OP_USERNAME, value.to_string_lossy()).unwrap();
  }
```

?

Reply:
could be, but I don't have a strong reason to not copy the existing style. (All `env::var()` could be changed to lossy conversion because ignoring non-utf-8 values doesn't make sense either.)

Reply:
This is unchanged for consistency with the other `env::var()` calls.